tests/test_main.py

```diff
@@ -54,6 +54,13 @@ def test_notify_response_error_returns_correctly(mock_request):
     assert response == ("notify request failed", 403)
 
 
+@responses.activate
+def test_notify_response_error_connection_reset_error(mock_request):
+    responses.add(responses.GET, url, body=Exception(ConnectionResetError))
```

Comment:
If the change mentioned above is made this should be:
```suggestion
    responses.add(responses.GET, url, body=ConnectionError)
```